src/BaseTemplates/ClassLibrary/project.json

```diff
@@ -2,12 +2,10 @@
   "version": "1.0.0-*",
```

Comment:
Can we make this 1.0.0.0.  Having the `*` breaks the ability for builds to be deterministic.